tests/request-test.js

```diff
@@ -0,0 +1,95 @@
+const assert = require('assert');
+const nock = require('nock');
+
+const request = require('../request');
+
+describe('request', function () {
+  beforeEach(function () {
+    nock.disableNetConnect();
+    nock.enableNetConnect('127.0.0.1');
+  });
+
+  afterEach(function () {
+    nock.enableNetConnect();
+    nock.cleanAll();
+  });
+
+  it('makes a network request and returns a promise', async function () {
+    const scope = nock('https://site.com')
```

Comment:
Suggest using `example.com` for test/placeholder purposes like this. Avoids unintended behaviors should the code somehow become "live" (the [corp.com domain](https://krebsonsecurity.com/2020/02/dangerous-domain-corp-com-goes-up-for-sale/) being a recent example).

Reply:
Went for `localhost` instead